packages/nextjs/.gitignore

```diff
@@ -34,4 +34,7 @@ yarn-error.log*
 .env.production.local
 
 # typescript
-*.tsbuildinfo
\ No newline at end of file
+*.tsbuildinfo
+
+# database
+services/database/seed.data.ts
```

Comment:
Ohh we need to remove current seed.data.ts file : 

```
git rm -r --cached packages/nextjs/services/database/seed.data.ts
``` 

And then commit it 🙌

Reply:
Oops, missed that. Thanks!! https://github.com/BuidlGuidl/SpeedRunEthereum-v2/pull/112/commits/c82d0ef64c4ce61873982de9f8dce9b4a77ee7a2

Reply:
Oh, It's detecting it as moving. It was confusing (no deleted files) but it should be good!

![image](https://github.com/user-attachments/assets/26c4cef2-e81f-458c-b2d2-0c61fdbd6fa6)